packages/wp-source/src/libraries/handlers/__tests__/author.tests.ts

```diff
@@ -69,6 +69,44 @@ describe("author", () => {
     ]);
   });
 
+  test("overwrites the data when fetched with { force: true }", async () => {
```

Comment:
Shouldn't be a test like this in all the other handlers?

Reply:
My thinking was that it's clear that the same behaviour copies over to the other handlers because we are using `Object.assign()` everywhere. 

But let me know if you still think we need them and I ll add them 🙂

Reply:
That's not the way to think about this. If someone refactors this code in the future, they will see the failing test in `author`, they will fix it, they'll see all tests passing and they can assume everything is fine in the rest of the handlers, they will ask for a review and unless someone is really paying attention to that small detail in the review, thing that I really doubt it, we would have introduced 3 bugs. So please add a test for each place where we need to ensure that this behavior is not changed over time.

- https://github.com/frontity/frontity/blob/8f764ad5431a70d53ebded630fa9021b4b2b547e/packages/wp-source/src/libraries/handlers/date.ts#L48
- https://github.com/frontity/frontity/blob/8f764ad5431a70d53ebded630fa9021b4b2b547e/packages/wp-source/src/libraries/handlers/postType.ts#L31
- https://github.com/frontity/frontity/blob/8f764ad5431a70d53ebded630fa9021b4b2b547e/packages/wp-source/src/libraries/handlers/postTypeArchive.ts#L43

Reply:
You're right about this 😓